app/models/series.rb

```diff
@@ -1022,10 +1022,34 @@ def increment_dependency_depth
     end
   end
 
+  def Series.reload_with_dependencies(id)
+    Series.find(id).reload_with_dependencies
+  end
+
+  def reload_with_dependencies
+    logger.info { "reload_with_dependencies: series #{id} (#{name}): start" }
+    result_set = [self.id]
+    next_set = [self.id]
+    until next_set.empty?
+      logger.debug { "reload_with_dependencies: next_set is #{next_set}" }
+      qmarks = next_set.count.times.map{ '?' }.join(',')
+      new_deps = Series.find_by_sql [<<~SQL, next_set].flatten ## this is so wackt that it must be done this way :(
```

Comment:
Can you add a link to the relevant docs so we know "why" it has to be done this way when we come back to this 6 months from now?